### PR TITLE
[#251] model healthcheck

### LINKED ISSUE #251: Add readiness check for model deployments

### PR DESCRIPTION
**Pull request by alexander-semenets**
fixes #251

### CODE REVIEW

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
ci test this please

File: examples/Test-Summation/Jenkinsfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ node {
         }
 
         stage('deploy'){
-            legion.deploy()
+            legion.deploy(timeout: 15)
```
Comment (akharlamov) [inline examples/Test-Summation/Jenkinsfile]: Maybe this should be liveness and readiness timeouts instead of a single timeout?

Comment (alexander-semenets) [reply examples/Test-Summation/Jenkinsfile]: added separate property for readiness probe

**Comment by alexander-semenets** [comment]
CI test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
ok to test

**Comment by alexander-semenets** [comment]
ok to test

**Pull request merged**